SynNoMoreEasyEnemies/Program.cs

```diff
@@ -51,28 +52,29 @@ out LazySettings
             foreach (var gmst in state.LoadOrder.PriorityOrder.OnlyEnabled().GameSetting().WinningOverrides()) {
                 // We only care about game settings that are floats
                 if (gmst is not IGameSettingFloatGetter floatGmst) continue;
+                if (floatGmst.EditorID == null) continue;
                 if (floatGmst.Data == null) continue;
 
                 // Only for our interesting settings
-                if (gmst.EditorID?.Contains("fLeveledActorMultEasy") == true) {
+                if (floatGmst.EditorID.Contains("fLeveledActorMultEasy")) {
```

Comment:
It's not, necessarily.  I was in the area simplifying the EditorID calls, and just swapped it over on a whim.   Generally since we've "scoped" the `gmst` to a more specific type `floatGmst`, that's usually the more informative variable that has more of the stuff that we want to access.  Since other lines will presumably be accessing `floatGmst` for its goodies, might as well have all the lines access that one for consistency.  `gmst` was only interesting up until we found out it was a `floatGmst`.  Now that variable is the interesting one to interact with.